_assets/scss/modules/_projects.scss

```diff
@@ -45,6 +65,13 @@
   }
 }
 
+.more-projects {
+  .h2 {
+    line-height: 1em;
```

Comment:
Yeah. I assumed that was the reasoning...just I generally think it's bad practice in case some day the text becomes long enough to wrap to two lines. I'm talking to Sean today about the baseline.